site/docs/v1/tech/installation-guide/kickstart.adoc

```diff
@@ -275,6 +275,28 @@ In this example I have a license Id of `eb7244dc-5d8e-40cd-a005-70b116fbda31`.
 
 If you are using the community edition, you can omit this field.
 
+=== Timeouts
+
+[NOTE.since]
+====
+Available since 1.20.1
+====
+
+Kickstart works by making API calls. By default each API request has a read and a connect timeout, and the API call will fail if it does not succeed within the timeout period. 
+
+You can configure these timeouts with a different value. For example, if you have high latency between the FusionAuth instance and its database, you may need to increase these values.
+
```

Comment:
Should we mention that in nearly all cases, 5 seconds should be very adequate, and in some (strange?) cases you may need to increase it. We added this for a client - and I'm not sure yet this is the problem, but adding this will allow them to verify it is not FusionAuth.

Reply:
Sure, made a note.